modules/core/src/main/scala/org/scalasteward/core/application/ExitCodePolicy.scala

```diff
@@ -28,5 +28,5 @@ object ExitCodePolicy {
 
   val SuccessIfAnyRepoSucceeds: ExitCodePolicy = successIf(_.successRepos.nonEmpty)
 
-  val SuccessOnlyIfAllReposSucceed: ExitCodePolicy = successIf(_.reposWithFailures.nonEmpty)
+  val SuccessOnlyIfAllReposSucceed: ExitCodePolicy = successIf(_.reposWithFailures.isEmpty)
```

Comment:
Good catch! Sorry I didn't test that with https://github.com/scala-steward-org/scala-steward/pull/3369 - thanks for fixing it!